test/testBrowserLanguage.js

```diff
@@ -0,0 +1,139 @@
+import browserLanguage from '../src/nginxconfig/util/browser_language';
+
+class MockLocales{
+    static languages = [];
+
+    static language = null;
+
+    static IntlBackup = null;
+
+    static navigator = null;
+
+    static setNavigatorLanguages(langs){
+        MockLocales.languages = langs;
+        return this;
+    }
+
+    static setNavigatorLanguage(lang){
+        MockLocales.language = lang;
+        return this;
+    }
+
+    static setDateTimeLocale(locale) {
+        const newDateTimeForMat = new Intl.DateTimeFormat(locale);
+        MockLocales.IntlBackup = Intl;
+        if (!locale){
+            // eslint-disable-next-line no-global-assign
+            Intl = undefined;
+            return this;
+        }
+
+        // eslint-disable-next-line no-global-assign
+        Intl = {
+            DateTimeFormat(){
+                return newDateTimeForMat;
+            },
+        };
+        return this;
+    }
+
+    static restoreDateTimeLocale(){
+        // eslint-disable-next-line no-global-assign
+        Intl = MockLocales.IntlBackup;
+        MockLocales.IntlBackup = null;
+        return this;
+    }
+
+    static setNavigator(navigator)
+    {
+        MockLocales.navigator = window.navigator;
+        window.navigator = navigator;
+        return this;
+    }
+
+    static restoreNavigator(){
+        window.navigator = MockLocales.navigator;
+        MockLocales.navigator = null;
+        return this;
+    }
+}
+Object.defineProperty(window.navigator,'languages',{get:() => {
+        return MockLocales.languages ?? [];
+    }});
```

Comment:
```suggestion
}});
```
Correct indentation (none)